### DISCUSSION TITLE
modoboa mails

### DISCUSSION THREAD

**Discussion by gisdinhoo** [modoboa/modoboa#2146] [Q&A]
Hello! Community, I want to know if there is a way to read emails stored on modoboa mail server.
I want to write a scrip that use incoming messages to notify users ( with SMS that contain mail object ) when a new message comes. It's for school work. Thanks!!

**Comment by tonioo** [comment]
@gisdinhoo One possible option is to use dovecot push notification framework (https://doc.dovecot.org/configuration_manual/push_notification/).
Another is to use a sieve script, see https://wiki2.dovecot.org/Pigeonhole/Sieve/Plugins/Extprograms.
